dev/tests/verification/TestModule/Suite/functionalSuiteHooks.xml

```diff
@@ -13,10 +13,13 @@
         </include>
         <before>
             <amOnPage url="some.url" stepKey="before"/>
-            <createData entity="createThis" stepKey="create">
+            <createData entity="createEntityOne" stepKey="createOne">
                 <field key="someKey">dataHere</field>
             </createData>
-            <click stepKey="clickWithData" userInput="$create.data$"/>
+            <createData entity="createEntityTwo" stepKey="createTwo">
+                <requiredEntity createDataKey="createEntityOne"/>
```

Comment:
Can we add two requiredEntities in the verification test?

Reply:
done.